lightgbm/src/main/scala/com/microsoft/azure/synapse/ml/lightgbm/params/BaseTrainParams.scala

```diff
@@ -20,6 +20,8 @@ abstract class BaseTrainParams extends Serializable {
   def seedParams: SeedParams
   def categoricalParams: CategoricalParams
 
+  def getNumClass: Int = 1
```

Comment:
Is this hard-coding intentional?

Reply:
yes, it gets overridden by a multiclass parameters